srs_data_management/config.yaml.example

```diff
@@ -41,3 +35,13 @@ sampling:
   data_count: '1200'
   tagger_count: '2'
   splits: '3'
+metrics:
+  metric:
+    baseline: '20171004'
+    evaluation:
+    - taglevel: '3'
+      releases:
+      - SprintSRS-2.13.1
+      - SprintSRS-2.12.0
+      - SprintSRS-2.11.0
+  output_dir: /Users/lisa/ASAPPinc/reports/spear
```

Comment:
newline (you should get a thing for your editor to auto-put these in here)

Reply:
good idea, just enabled it for pycharm